scripts/post-vsts-artifact-comment.mjs

```diff
@@ -44,26 +49,65 @@ async function main() {
 }
 \`\`\`
 and then running \`npm install\`.
-`,
-    });
+`;
 
     // Temporarily disable until we get access controls set up right
     // Send a ping to https://github.com/microsoft/typescript-make-monaco-builds#pull-request-builds
-    await gh.request("POST /repos/microsoft/typescript-make-monaco-builds/dispatches", { event_type: process.env.SOURCE_ISSUE, headers: { Accept: "application/vnd.github.everest-preview+json" } });
+    await gh.request("POST /repos/microsoft/typescript-make-monaco-builds/dispatches", { event_type: `${SOURCE_ISSUE}`, headers: { Accept: "application/vnd.github.everest-preview+json" } });
+
+    return comment;
 }
 
-main().catch(async e => {
+let newComment;
+let emoji;
+
+try {
+    newComment = await main();
+    emoji = "✅";
+}
+catch (e) {
     console.error(e);
-    process.exitCode = 1;
-    if (process.env.SOURCE_ISSUE) {
-        const gh = new Octokit({
-            auth: process.argv[2],
-        });
-        await gh.issues.createComment({
-            issue_number: +process.env.SOURCE_ISSUE,
-            owner: "Microsoft",
-            repo: "TypeScript",
-            body: `Hey @${process.env.REQUESTING_USER}, something went wrong when looking for the build artifact. ([You can check the log here](https://typescript.visualstudio.com/TypeScript/_build/index?buildId=${process.env.BUILD_BUILDID}&_a=summary)).`,
-        });
-    }
+    newComment = `Hey @${REQUESTING_USER}, something went wrong when looking for the build artifact. ([You can check the log here](https://typescript.visualstudio.com/TypeScript/_build/index?buildId=${BUILD_BUILDID}&_a=summary)).`;
+    emoji = "❌";
+}
+
+const resultsComment = await gh.issues.createComment({

```

Comment:
This is more or less the same as https://github.com/microsoft/DefinitelyTyped-tools/blob/main/packages/dtslint-runner/src/post-results.ts#L67.